test/init_local_test.go

```diff
@@ -123,6 +123,28 @@ repo_b
 			{"repository_id": "repo_b"},
 		}, rows)
 	})
+
+	// Test bblfsh queries through superset
+	s.T().Run("superset-bblfsh", func(t *testing.T) {
+		req := require.New(t)
+
+		lang, err := client.bblfsh("hello.js", `console.log("hello");`)
+		req.NoError(err)
+		req.Equal("javascript", lang)
+	})
+
+	// Test gitbase can connect to bblfsh with a SQL query that uses LANGUAGE
```

Comment:
`LANGUAGE` function in gitbase doesn't use bblfsh

Reply:
true, I forgot it uses enry directly. I'll redo the query asking for UAST.

Reply:
done in 76dc0a9